editor/src/core/shared/element-template.ts

```diff
@@ -961,6 +960,32 @@ export function isRegularJSXAttribute(attribute: ModifiableAttribute): attribute
   )
 }
 
+export function modifiableAttributeToValuePath(
```

Comment:
could you add some test coverage for this? I _think_ this function will be key for the data tracing features next week

Reply:
Yup.